public.js

```diff
@@ -253,6 +253,45 @@ var init = function(app, schema, sequelize, config) {
   app.get('/js/:domain*', js_result('success'));
   app.get('/jsaction/:domain*', js_result('action'));
 
+  function json_result (successMetric) {
```

Comment:
This technically works, but for consistency with the rest of the code and because this is technically inside the init function, let's go with `var json_result = function(successMetric) {...}`